python/composio/tools/env/host/shell.py

```diff
@@ -28,13 +33,42 @@ def __init__(self) -> None:
             text=True,
             bufsize=1,
         )
-        self.logger.debug(f"Initial data from session: {self.id} - {self._read()}")
+        self.logger.debug(
+            f"Initial data from session: {self.id} - {self._read(wait=False)}"
+        )
+
+    def _wait_for_cmd(self, cmd: str) -> None:
```

Comment:
where is this being used ?

Reply:
https://github.com/ComposioHQ/composio/blob/879ae3dbc2c9a8eec922a76e6c523436eca01c42/python/composio/tools/env/host/shell.py#L71